website/docs/ref/metro-transformer.mdx

```diff
@@ -0,0 +1,101 @@
+# Metro transformer
+
+[Metro bundler](https://metrobundler.dev/) is a JavaScript bundler used in React Native apps. This package offers an alternative to the [`lingui compile`](/ref/cli#compile) command: a transformer that enables Metro to compile `.po` files on the fly. The transformer enables you to `import` `.po` files directly, instead of running `lingui compile` and `import`ing the resulting JavaScript (or TypeScript) files.
+
+## Installation
+
+> Only Expo SDK 49 and React Native v0.72.1 or newer are supported.
```

Comment:
I think this could be declared as constraints in peerDependecies as well

Reply:
done